platform/components/docker-gbsyncd-broncos.mk

```diff
@@ -0,0 +1,37 @@
+LIBSAI_BRONCOS = libsaibroncos_0.0.1_amd64.deb
+$(LIBSAI_BRONCOS)_URL =
+
+ifneq ($($(LIBSAI_BRONCOS)_URL),)
+
+DOCKER_GBSYNCD_BRONCOS_STEM = docker-gbsyncd-broncos
+DOCKER_GBSYNCD_BRONCOS = $(DOCKER_GBSYNCD_BRONCOS_STEM).gz
+DOCKER_GBSYNCD_BRONCOS_DBG = $(DOCKER_GBSYNCD_BRONCOS_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_GBSYNCD_BRONCOS)_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
+
+$(DOCKER_GBSYNCD_BRONCOS)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_BUSTER)
```

Comment:
Please update base docker from buster to bullseye like https://github.com/Azure/sonic-buildimage/pull/11288.